.github/workflows/update-lockfile.yml

```
@@ -45,7 +45,7 @@ jobs:
           git config user.name "GitHub"
           git config user.email "noreply@github.com"
           git add lockfile.json
-          UPDATED_PARSERS=$(echo $(/tmp/jd -set /tmp/old_lockfile.json lockfile.json | grep @ | sed 's/","revision"\]//' |  sed 's/@ \["//') | sed 's/ /, /g')
+          UPDATED_PARSERS=$(echo $(/tmp/jd -set /tmp/old_lockfile.json lockfile.json | grep @ | sed 's/","revision"\]//' | sed 's/"\]//' |  sed 's/@ \["//') | sed 's/ /, /g')
```

This can be simplified (and maybe even further but I'm not a `sed` expert):
```suggestion
          UPDATED_PARSERS=$(/tmp/jd -set /tmp/old_lockfile.json lockfile.json | sed -n '/@/s/","revision"]//;s/"]//;s/@ \["//p' | sed 'N;s/\n/, /g')
```

---

Is this really simpler? I've tried to avoid any complex commands and rather go with many simple that I was sure that they won't cause any errors on strange input. I don't understand what you're doing here since I don't know about any `sed` flags.

If your suggestion works I can blindly accept it. Did you test it?

---

> Is this really simpler?

It's simpler in the sense that it runs 3 commands instead of 7.
You can ignore it if you find it complicated.

> I don't understand what you're doing here

The first `sed` matches lines that contain `@` and runs the substitutions on those.
The `-n` argument and `p` flag make it so only the matching lines are printed.

The second `sed` uses the `N` command to read the next line so that the `\n` substitution works.

> If your suggestion works I can blindly accept it. Did you test it?

I did test it locally and it works but I don't know if it has any edge cases.